app.js

```diff
@@ -1,12 +1,24 @@
 const express = require("express");
-const { getCategories } = require("./controllers/controllers");
+const { getCategories, getReviewById } = require("./controllers/controllers");
 
 const app = express();
 
 app.get("/api/categories", getCategories);
 
+app.get("/api/reviews/:review_id", getReviewById);
+
 app.all("/*", (req, res) => {
   res.status(404).send({ msg: "Endpoint does not exist" });
 });
 
+// Error-handling middleware functions
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Please enter a valid ID" });
```

Comment:
It's worth bearing in mind that these errors may not be being delivered directly to the user, but to front-end devs that will be responsible for displaying the error message. Therefore, it would be more logical to rephrase this - though it's not bad as it is.